Example/HCaptcha_Tests/Core/HCaptcha__Config__Tests.swift

```diff
@@ -47,4 +115,21 @@ class HCaptcha__Config__Tests: XCTestCase {
             of: "some-api-key.ios-sdk.hcaptcha.com",
             with: host))
     }
+
+    func test__Custom__Theme() {
+        let cusomtTheme = """
+          {
+            primary: {
+              main: "#00FF00"
+            },
+            text: {
+              heading: "#454545",
+              body   : "#8C8C8C"
+            }
+          }
+        """
+        let config = createConfig(customTheme: cusomtTheme)
```

Comment:
```suggestion
        let config = createConfig(customTheme: customTheme)
```


.. how did no test break from this? :)

Reply:
Probably because the original var has typo too), will fix